### PR TITLE
More components

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: app/views/catalog/_show_sidebar.html.erb [hunk 1]
```diff
@@ -1,12 +1,2 @@
 <%= render 'show_tools' %>
-
-<% unless document.more_like_this.empty? %>
-  <div class="card">
-    <div class="card-header">More Like This</div>
-    <div class="card-body">
-      <%= render collection: document.more_like_this,
-                 partial: 'show_more_like_this',
```
Comment (mejackreed) [inline app/views/catalog/_show_sidebar.html.erb:8]: Do we leave this partial in for backwards compatibility?

Comment (cbeer) [reply app/views/catalog/_show_sidebar.html.erb:8]: Exactly. I suspect it's not long for the world, but might make it into 8.x.

File: lib/blacklight.rb [hunk 2]
```diff
@@ -70,6 +70,16 @@ def self.connection_config=(value)
     Blacklight::RuntimeRegistry.connection_config = value
   end
 
+  def defaults_version
```
Comment (jcoyne) [inline lib/blacklight.rb:73]: What calls this method?

Comment (cbeer) [reply lib/blacklight.rb:73]: Nothing yet. Just getting it out there so we can start opting in to some of the new behavior without needing a major release.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**